tests/signing/testdata/signing/TestSigningPayload/empty_model_file_shard

```diff
@@ -0,0 +1,21 @@
+{
+  "_type": "https://in-toto.io/Statement/v1",
+  "subject": [
+    {
+      "name": "empty_file_model",
+      "digest": {
+        "sha256": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+      }
```

Comment:
just to confirm this is the empty sha256 right?

Reply:
Yes, it has to be

```
[...]$ echo -n "" | sha256sum 
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  -
```